docs/source/testing/mocking.md

```diff
@@ -30,15 +30,19 @@ server.listen().then(({ url }) => {
 });
 ```
 
-> Note: If `typeDefs` has custom scalar types, `resolvers` must still contain the `serialize`, `parseValue`, and `parseLiteral` functions
+> Note: If `typeDefs` has any custom scalar types, your `resolvers` must contain the `serialize`, `parseValue`, and `parseLiteral` functions for each of those custom scalars.
 
-Mocking logic simply looks at the type definitions and returns a string where a string is expected, a number for a number, etc. This provides the right shape of result. By default, when using mocks, any existing resolvers are ignored. See the ["Using existing resolvers with mocks"](#using-existing-resolvers-with-mocks) section below for more info on how to change this behavior.
+Mocking logic looks at the types expected by your schema's fields and returns values based on those types. For example, mocking will return a string where a string is expected, a number for a Int, etc. The shape of your mocked results should always be consistent with the shape of a real result from a successful GraphQL operation.
 
-For more sophisticated testing, mocks can be customized to a particular data model.
+When using mocks your existing resolvers are **ignored** by default. See the ["Using existing resolvers with mocks"](#using-existing-resolvers-with-mocks) section below to see how to change this behavior.
```

Comment:
"By default when using mocks, any resolvers you've defined are **ignored**.

(I like to put "by default" at the front of sentences that describes a default behavior, just so that context is in the reader's head off the bat. Also enables the big bold word to appear at the end of the sentence for extra emphasis)

Reply:
"To configure this behavior, see [Using existing resolvers with mocks](...)"

* No need to put section name in quotation marks
* Slightly shorter than describing as "the ____ section"
* Puts link at end of sentence for improved clickability